setup.py

```diff
@@ -45,8 +46,9 @@
     },
     install_requires=requirements,
     extras_require={
-        'bucket-access': 'gsutil',
+        'bucket-access': ['gcsfs', 'google-cloud-storage'],
```

Comment:
is `google-cloud-storage` not a dependency of `gcsfs`?

Reply:
I had added it as an explicit dependency because I was using it explicitly to get the environment variable name used for the gcloud project name. No longer doing that, so I removed the dependency.

Reply:
Okay. thank you.